doc/en/api/1.0.0/gwcdiskquota.yaml

```diff
@@ -0,0 +1,50 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache DiskQuota
+  description: The REST API for Disk Quota management provides a RESTful interface through which clients can configure the disk usage limits and expiration policies for a GeoWebCache instance through simple HTTP calls.
+schemes:
+- http
+host: localhost:8080
+basePath: /geoserver/gwc/rest
+paths:
+  /diskquota:
+    get:
+      produces:
+        - application/xml
+        - application/json
+      operationId: diskQuotaGet
+      summary: Returns the gwc Quota configuration
+      description: Returns the GeoWebCache DiskQuota configurations which include disk usage limits and expritation policies for a gwc instance.
+      responses:
+        200:
+          description: OK
+          examples:
+            application/xml: |
+              <gwcQuotaConfiguration>
+                <enabled>true</enabled>
+                <diskBlockSize>2048</diskBlockSize>
+                <cacheCleanUpFrequency>5</cacheCleanUpFrequency>
+                <cacheCleanUpUnits>SECONDS</cacheCleanUpUnits>
+                <maxConcurrentCleanUps>5</maxConcurrentCleanUps>
+                <globalExpirationPolicyName>LRU</globalExpirationPolicyName>
+                <globalQuota>
+                  <value>100</value>
+                  <units>MiB</units>
+                </globalQuota>
+                <layerQuotas/>
+              </gwcQuotaConfiguration>
```

Comment:
In cases where we support multiple media types (e.g. XML and JSON) it is useful to have an example of both

Reply:
Addressed